### DISCUSSION TITLE
Unable to upload TAPP file to Athom plug

### DISCUSSION THREAD

**Discussion by p4mr** [arendst/Tasmota#19040] [Q&A] [answered]
I'm trying to add Partition Manager to an Athom plug (to resize the filesystem, so I can upload Qasim), and am failing at the first hurdle. I've gone to the Console (from the web browser), and issued the following command (which I believe is correct based on documentation I've read to date)

`UrlFetch http://raw.githubusercontent.com/arendst/Tasmota/development/tasmota/berry/modules/Partition_Manager.tapp`

This is immediately returning

stat/tasmota_8FA5DF/RESULT = {"Command":"Unknown"}

Current version of Tasmota is 12.5.0, which I believe supports the UrlFetch command.

What am I doing wrong ?

**Comment by barbudor** [comment, marked as answer]
As far as I know, Athom plugs are based on ESP8266
Tapp, Berry and the Partition managers are only for ESP32 based devices

**Reply by sfromis** [reply]
Yes, upgrading of ESP8266 devices is not affected, and with ESP32 devices it would only be in case of a much older install than 12.5.

**Comment by s-hadinger** [comment]
Alternative is just to download locally the tapp file from GitHub via a beowser, and use the file upload to the file-system. Then reboot.

**Reply by p4mr** [reply]
I've tried this, to no avail (see my original thread).

Basically, I want to install 13.0.0 on the plug, but it won't load, stating the file is too big (hence trying to use the Partition Manager to change the file size layout).

If I can't use TAPP, how do I get the plug to load 13.0.0 ?

**Reply by Noschvie** [reply]
Try loading the minimal firmware before

**Reply by sfromis** [reply]
Partition manager is not relevant to ESP8266 devices, but what **is** very relevant is to **always** use `.gz` files when upgrading. It will likely fail if you do not do that.

Unless you are building a custom binary, I'd suggest not worrying about "minimal", but just do an OTA upgrade to 
http://ota.tasmota.com/tasmota/release/tasmota.bin.gz
(Without downloading first)
